pkg/metrics/metrics.go

```diff
@@ -351,20 +405,66 @@ func (c *Counter) IsAlertFiring(alert string, checkedNS, ignoredNS []string) (bo
 	return false, nil
 }
 
-func getPromHost(c client.Client) (*string, error) {
+// GetServiceEndpoint accepts a client,namespace,svcName and portName and attempts to retrive
+// the services endpoint in the form of resolveable.service:portnumber.
+func GetServiceEndpoint(c client.Client, namespace, svcName, portName string) (string, error) {
```

Comment:
The ServiceEndpoint here is actually not an endpoint resource right? it is just the service+port. The function name is a bit confusing.

And I think it can be handled by `const` instead of function.

Reply:
This function name is confusing in the context of kubernetes. I will update it to `GetService`. I think we should keep as function rather then consts in-case ports change in future.